rdr_service/genomic/genomic_job_components.py

```diff
@@ -646,3 +651,159 @@ def _get_sex_at_birth(self, participant_id):
                           .first()
         return self._SEX_AT_BIRTH_CODES.get(
                  result[0].lower().split('_')[-1], 'NA') if result else 'NA'
+
+
+class ManifestDefinitionProvider:
+    """
+    Helper class to produce the definitions for each manifest
+    """
+    # Metadata for the various manifests
+    ManifestDef = namedtuple('ManifestDef', ["job_run_field",
+                                             "source_data",
+                                             "destination_bucket",
+                                             "output_filename",
+                                             "columns"])
+
+    def __init__(self, job_run_id=None, bucket_name=None,):
+        # Attributes
+        self.job_run_id = job_run_id
+        self.bucket_name = bucket_name
+
+        self.MANIFEST_DEFINITIONS = dict()
+        self._setup_manifest_definitions()
+
+    def _setup_manifest_definitions(self):
+        """
+        Creates the manifest definitions to use when generating the manifest
+        based on manifest type
+        """
+        # Set each Manifes Definition as an instance of ManifestDef()
```

Comment:
```suggestion
        # Set each Manifest Definition as an instance of ManifestDef()
```

Reply:
I'll update this in the next PR I have for the CVL Array manifests, probably on Monday.